torch/csrc/distributed/rpc/init.cpp

```diff
@@ -239,6 +239,15 @@ PyObject* rpc_init(PyObject* /* unused */) {
               "_deserialize",
               &PyRRef::unpickle,
               py::call_guard<py::gil_scoped_release>())
+          .def(
+              "_get_future",
```

Comment:
This LGTM. 

(Doesn't need to be added to this PR) With this future, does it mean we no longer need `confirmedByOwner_` var any more? Instead we can check if the future is completed? There might be a tiny race that the `confirmPendingUser` on the future hasn't fired yet, so that the UserRRef is still pending in the RRefContext even if the future is completed though.

Reply:
Given this, should we keep `confirmedByOwner_`? Also, it looks like there is a public API in `rpc/init.cpp` that uses confirmedByOwner_ so it may be challenging to deprecate this.